3-async-await/solution.js

```diff
@@ -20,16 +20,70 @@ Example:
 9. as extra challenge: add Promise.race() and Promise.any(), and try to get the idea of what happens
 */
 
-function solution() {
+const products = require('./products');
+const prices = require('./prices');
+
+function getId() {
+    return Number(String(Date.now()).slice(-2));
+}
+
+async function solution() {
     // YOUR SOLUTION GOES HERE
 
     // You generate your id value here
+    const id = getId();
+    try {
+        // You use Promise.all() here
+        // You use Promise.allSettled() here
+        // Log the results, or errors, here
+        const [product, price] = await Promise.all([products(id), prices(id)]);
```

Comment:
You could have wrapped this and the `Promise.all` related code in a try catch, so the rest of the code works even if this one fails